pkg/virt-operator/resource/apply/reconcile.go

```diff
@@ -716,20 +716,23 @@ func (r *Reconciler) Sync(queue workqueue.RateLimitingInterface) (bool, error) {
 		return false, err
 	}
 
+	commonInstancetypesStart := time.Now()
```

Comment:
I wonder if we can drop b57f1e4cfb7656703cc877b6db72ff28dce3fa9d now

Reply:
Dropped.

Reply:
FWIW given these are trace level logs I think they are perfectly acceptable even with the fix.